CHANGELOG.md

```diff
@@ -56,6 +56,45 @@ All notable changes to this project will be documented in this file.
             * `get_ext_cal_last_temp`
             * `get_self_cal_last_date_and_time`
             * `get_self_cal_last_temp`
+        * Measurement library methods. - [#806](https://github.com/ni/nimi-python/issues/806)
+            * `add_waveform_processing`
+            * `clear_waveform_measurement_stats`
+            * `clear_waveform_processing`
+            * `fetch_measurement`
+            * `read_measurement`
```

Comment:
Are there system tests for all these?

Reply:
Good point. I'd only added the metadata and any changes needed to build properly. I can add tests for some of these from the other PR branch, but not all of them -- specifically, clear_waveform_measurement_stats needs to have fetch_measurement_stats available in order to be properly tested.

Reply:
I think you should add the tests in this same PR in which the things are added/made public. Keep an eye on code coverage numbers too. It may allow us to bump pass/fail thresholds.

Reply:
Done. As mentioned, clear_waveform_measurement_stats can't effectively be tested until fetch_measurement_stats exists, but all other system tests were added.